exercises/concept/inventory-management/.docs/hints.md

```diff
@@ -4,32 +4,32 @@
 
 - [The Python Dictionary Tutorial](https://docs.python.org/3/tutorial/datastructures.html#dictionaries) can be a great introduction.
 
-## 1. Create the dictionary
+## 1. Create an inventory based on a list
 
 - You need a [for loop](https://docs.python.org/3/tutorial/controlflow.html#for-statements) to iterate the list of items, then insert each item in the dictionary if missing and increment the item count using the dictionary accessor.
 - You can use [`setdefault`](https://www.w3schools.com/python/ref_dictionary_setdefault.asp) to make sure the value is set before incrementing the count of the item.
 - This function should [return a dict](https://www.w3schools.com/python/ref_keyword_return.asp).
 
-## 2. Add items to a dictionary
+## 2. Add items from a list to an existing dictionary
```

Comment:
The item may be a single item and not part of a list, so this is prescriptive and may not be accurate.